src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,100 @@
+package wolox.training.models;
+
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import wolox.training.exceptions.users.BookAlreadyOwnedException;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(nullable = false)
+    private String username;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private LocalDate birthday;
+
+    @ManyToMany(cascade = CascadeType.ALL)
+    @JoinTable(name = "book_user",
+            joinColumns = @JoinColumn(name = "book_id", referencedColumnName = "id"),
+            inverseJoinColumns = @JoinColumn(name = "user_id", referencedColumnName = "id"))
```

Comment:
`@JoinTable` annotation is not necessary if you don't want to redefine the columns, the ManyToMany annotation generates a `join table` by default 😃

Reply:
You're right! It keep working!
![image](https://user-images.githubusercontent.com/19678474/80401620-41d7a900-8882-11ea-95a3-7ad4869f3b0b.png)